docs/platform/tutorials/image-processing/web-ui.md

```diff
@@ -1,31 +1,137 @@
-# 7. Deploy the web UI
+# Web UI
 
-In this part of the tutorial you add a service to provide a simple UI with which to monitor and control the pipeline.
+In this part of the tutorial you learn about the web UI service. 
 
-The following screenshot shows the last image processed from one of the video streams, as well as the map with a count of all the objects detected so far, and their location:
+![Web UI pipeline](../image-processing/images/web-ui-pipeline-segment.png)
+
+This provides the rather fancy interface for you to interact with this project.
+
+The following screenshot shows vehicle density at various points in London:
 
 ![image processing web UI](./images/web-ui.png)
 
-!!! tip
+## 💡 Key ideas
+
+The key ideas on this page:
+
+* How a web client can read data from a Quix topic using Quix Streaming Reader API
+* WebSockets as a way of streaming data into a web client
+* Microsoft SignalR is the WebSockets technology used in the Streaming Reader API
+* Access an external web application from the pipeline view
+
+## What it does
+
+The key thing this service does is provides a UI that enables you to see vehicle data in real time, displayed on a Google map.
+
+The UI is a JavaScript web client. The most important thing to understand is how this service obtains data from the Quix pipeline. This is done through use of the [Quix Streaming Reader API](../../../apis/streaming-reader-api/intro.md). 
```

Comment:
Excuse my nitpicking, but isn't it now based on Angular?... which is a...quote "a TypeScript-based, free and open-source single-page web application framework"

Reply:
Yes, quite right. Fixed.